src/file-manager/fm-properties-window.c

```diff
@@ -3685,13 +3690,20 @@ create_basic_page (FMPropertiesWindow *window)
 
 	icon_pixmap_widget = create_image_widget (
 		window, should_show_custom_icon_buttons (window));
+#if GTK_CHECK_VERSION (3, 0, 0)
+	gtk_widget_set_valign (icon_pixmap_widget, GTK_ALIGN_START);
+	gtk_widget_show (icon_pixmap_widget);
+
+	gtk_box_pack_start (GTK_BOX (hbox), icon_pixmap_widget, FALSE, FALSE, 0);
+#else
 	gtk_widget_show (icon_pixmap_widget);
 
 	icon_aligner = gtk_alignment_new (1, 0, 0, 0);
```

Comment:
The first argument here means xalign is at the end... guess it should be done with halign in GTK+3 code.